libs/ts-rest/react-query/src/lib/react-query.ts

```diff
@@ -106,23 +137,13 @@ type DataReturnQueries<
 type DataReturnInfiniteQuery<
   TAppRoute extends AppRoute,
   TClientArgs extends ClientArgs
-> = AreAllPropertiesOptional<
-  Without<DataReturnArgs<TAppRoute, TClientArgs>, never>
-> extends true
-  ? (
-      queryKey: QueryKey,
-      args?: (
```

Comment:
Removed the optional version as it makes no sense to use an infinite query with no args, as you'd need to at least map your page number to some arg. Technically a breaking change, but extremely unlikely that anyone is doing this.